data/json/items/tool/cooking.json

```
@@ -629,8 +629,7 @@
   {
     "id": "mortar_pestle",
     "type": "TOOL",
-    "name": "mortar and pestle",
-    "name_plural": "sets of mortar and pestle",
+    "name": { "str": "mortar and pestle", "str_pl": "sets of mortar and pestle" },
```

```suggestion
    "name": { "str": "mortar and pestle", "str_pl": "mortars and pestles" },
```
The original sounds awkward. There is a consensus plural for this term already.